R/model.R

```diff
@@ -352,14 +352,19 @@ CmdStanModel <- R6::R6Class(
 #'
 NULL
 
-compile_method <- function(quiet = TRUE,
+generate_arg_strings_method <- function(quiet=TRUE,
+                           make_or_stanc=list('make','stanc'),
                            dir = NULL,
                            include_paths = NULL,
                            cpp_options = list(),
                            stanc_options = list(),
-                           force_recompile = FALSE,
                            #deprecated
                            threads = FALSE) {
+
+  if (!is.null(make_or_stanc)) {
```

Comment:
Is there a use case for it ever being NULL or can we error if it's NULL?

Reply:
Throwing an error when NULL seems appropriate; I was copying the template for how other functions used checkmate, but especially if we make this a private function, it should never be NULL.

Reply:
Yeah in some cases NULL is OK so we let it through but if it's not OK here then I think in that case you can just remove the if clause. I'm pretty sure the checkmate assert function will error by default if NULL.